config/routes.rb

```diff
@@ -184,6 +184,8 @@ def scoped_navigation_routes(context, navigation)
           mount DelayedJobWeb => "/delayed_job"
         end
 
+        resource :forced_password_resets, only: [:edit, :update]
```

Comment:
A few lines down there are a number of routes nested within `users`. It would be nicer if this were next to those in this file. When doing that, it's best to use nested namespaces in the Ruby code -- rename the controller to e.g. `Hub::Users::XyzController`.